.circleci/config.yml

```diff
@@ -72,7 +72,7 @@ attach-to-workspace: &attach-to-workspace
   attach_workspace:
     at: .
 
-system-builder-ruby24: &system-builder-ruby24
+system-builder-latest: &system-builder-latest
```

Comment:
i think i prefer the explicit `ruby24` tag than the more generic `latest` 🤔

Reply:
Agree. `Latest` gives the impression that we will run into the latest version of ruby